config.reek

```diff
@@ -20,6 +20,13 @@ LongParameterList:
   exclude:
   - initialize
 
+UncommunicativeMethodName:
+  accept:
+    - s3 # elb, rds, ec2, sqs are all fine
+UncommunicativeVariableName:
+  accept:
+    - s3 # elb, rds, ec2, sqs are all fine
```

Comment:
I didn't find any usages in those repo's I'm concerned about.  

But I do think we should update the comments that you added to `config.reek`.  The comment says `ec2` is a valid name, but without adding it to the `accept` list, I don't think it is valid.